tools/scoring/score/parse_config.py

```diff
@@ -177,3 +177,57 @@ def retrieve_reporting_translations(
       }
 
     return translations
+
+  @staticmethod
+  def aggregate_scores_nondbo(
+      # TODO: create Dimension and Score types to replace generics
```

Comment:
can you please create those first, it will help me understand what they are :)

Reply:
https://github.com/google/digitalbuildings/pull/468/commits/cfe49ce45c291cf1421018d3baf5962616179d38

Reply:
The `Dimension` type(s) won't add much beyond reference convenience as they each only take two arguments; I'd like to punt on those until at least one piece of the core functionality is landed.